docs/scos/dev/feature-integration-guides/202200.0/multiple-abstract-products-promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,190 @@
+Make sure that in the database the configured data are added to the `spy_glossary` table.
+
+{% endinfo_block %}
+
+### 4) Add Zed translations
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that all labels and help tooltips in the discount form has English and German translation.
+
+{% endinfo_block %}
+
+### 5) Set up behavior
+
+Set up the following behaviors:
+
+| PLUGIN                                                      | SPECIFICATION                                                        | PREREQUISITES | NAMESPACE                                                                                                            |
+|-------------------------------------------------------------|----------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------------------------|
+| DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin | Adds discount promotion form name postfix to the Add To Cart form.   | None          | SprykerShop\Yves\DiscountPromotionWidget\Plugin\CartPage\DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin |
+
+**src/Pyz/Yves/CartPage/CartPageDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\CartPage;
+
+use SprykerShop\Yves\CartPage\CartPageDependencyProvider as SprykerCartPageDependencyProvider;
+use SprykerShop\Yves\DiscountPromotionWidget\Plugin\CartPage\DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin;
+
+class CartPageDependencyProvider extends SprykerCartPageDependencyProvider
+{
+    /**
+     * @return array<\SprykerShop\Yves\CartPageExtension\Dependency\Plugin\AddToCartFormWidgetParameterExpanderPluginInterface>
+     */
+    protected function getAddToCartFormWidgetParameterExpanderPlugins(): array
+    {
+        return [
+            new DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the plugin works correctly:
+
+1. [Create a discount](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-cart-rules.html) and select **Promotional Product** radio button in **Discount Calculation** form.
+2. Add a product abstract sku into the **Abstract Product SKU(s)** field.
+3. Create another discount with the at least one same **Promotional Product**.
+4. To fulfill the discounts' requirements, add items to the cart.
+5. Make sure that both discount are displayed in the the **Promotional Product** section on the cart page.
+
+{% endinfo_block %}
+
+
+### 4) Build Zed UI frontend
+
+Enable Javascript and CSS changes:
+
+```bash
+console frontend:zed:install-dependencies
+console frontend:zed:build
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that you can create a discount with multiple promotional products:
```

Comment:
Isn't this the same as the plugin verification process? Do we need this?

Reply:
@JacquieRooney, basically - yes. However, when we verify that the plugin works, our goal is to make sure that all promotional products are displayed on the cart page. While here we have to make sure that a discount is applied after a promotion product is added to the cart.
Should this step be added to the plugin verification instruction? So we can remove this section.

Reply:
No, we can leave it as-is. I understand the difference between the two exercises now.